tests/test_download.py

```diff
@@ -124,7 +124,7 @@ def test_incorrect_snapshot_order_news(self):
             start_snapshot = "2021-10"
             urls = get_common_crawl_urls(start_snapshot, end_snapshot, news=True)
 
-    @pytest.mark.flaky(reruns=4)
+    @pytest.mark.flaky(reruns=5)
```

Comment:
Honestly, while you're here, if you don't mind deleting this test for now I think it's for the best. The flaky hasn't been much help. We can mock it later, but it's more hassle than it's worth at this point.

Reply:
Sure, can remove in a separate PR.

Reply:
Opened https://github.com/NVIDIA/NeMo-Curator/pull/235.